packages/extension-ui/src/Popup/CreateAccount/index.tsx

```diff
@@ -94,17 +97,16 @@ function CreateAccount(): React.ReactElement {
             isBusy={isBusy}
             onBackClick={_onPreviousStep}
             onCreate={_onCreate}
-            onNameChange={setName}
             setGenesis={setGenesis}
           />
         )}
+        {step === 4 && <AccounCreationSuccess />}
```

Comment:
`isLastStep`

Reply:
Not necessarily - I kind of wanted to follow the literal numeration pattern that goes from 0 up to the end. I added the `isLastStep` const for a slightly different purpose (well, at least semantically, cause technically it's of course the same 🙂 ). Lmk wdyt.